README.md

```diff
@@ -74,8 +74,11 @@ fuego get mycollection mydocumentid
 
 If you need to use fuego with the firestore emulator instead of a real firestore
 database, set the FIRESTORE_EMULATOR_HOST environment variable to something
-appropriate (usually, localhost:8080). if no project is set, fuego will default
-to 'default'.
+appropriate (usually, localhost:8080).
+
+### Firebase project ID
+
+Set the project ID with `-projectid NAME`. Defaults to 'default'.
```

Comment:
uhm, the problem is that if you are not using the firestore emulator, you are likely using a Google Applications Credentials file, where a project id is specified. Fuego will actually default to that value. I think that's why I left that part in the firestore emulator section, since that's where we are likely not to use a google application credentials file.

Reply:
I'will keep the first part (Set the proje...) and perhaps clarify the second.

Reply:
```suggestion
Set the project ID with `-projectid NAME`, otherwise fuego will fallback to the project defined by the GOOGLE_APPLICATION_CREDENTIALS file or to 'default', if none.
```